### ISSUE TITLE
Devops developer

### ISSUE THREAD

**Issue by hamidmaddi** [rajasegar/ember-responsive-tabs#4]
The addon only works when the page is fully opened, it loses tab names when reducing window size.
![image](https://user-images.githubusercontent.com/5750045/34313068-9afd55c4-e736-11e7-9748-30e4b71530fd.png)

**Comment by rajasegar** [comment]
@hamidmaddi I think it will look for  icons other than text when the size is crossing a particular breakpoint, can we add some icons to the text as well and check

**Comment by hamidmaddi** [comment]
I did try with icons, I will post an image tomorrow.

Sent from my iPhone

On Dec 25, 2017, at 10:45 PM, Rajasegar Chandran <notifications@github.com<mailto:notifications@github.com>> wrote:


@hamidmaddi<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fhamidmaddi&data=02%7C01%7C%7C3d5495dadc8144808a5e08d54c1316bb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636498567216497161&sdata=bvpsbbuznbc9IxlA1k2mBfgri7eARHB1kL9gp55RgD8%3D&reserved=0> I think it will look for icons other than text when the size is crossing a particular breakpoint, can we add some icons to the text as well and check

**Comment by hamidmaddi** [comment]
The tabs have issues even with icons. I love this project and would like to be able to use the tabs without icons., is that possible?
![image](https://user-images.githubusercontent.com/5750045/34347241-30c79eee-e9cf-11e7-853b-5205903c3992.png)

**Comment by rajasegar** [comment]
@hamidmaddi Yes, that is possible, We can hack at the css if you want the tabs without icons but what you would want to display in the place of icons, considering the space available we can have only one or two letter words there. 
In addition to that , the styles need to be cleaned up for the addon a little bit. I can't promise you anything about the timeline when it will be ready.

**Comment by hamidmaddi** [comment]
I will try to spend some time on it. I did hack the css as you say to display the tabs on the left side.

**Comment by sjmaceyful** [comment]
Not sure where this is at, but an idea might be to strip back the tabs to very basic layout styling. Could also look at allowing block content in the tab buttons so they can be fully customised.
